flexget/plugins/output/hipchat.py

```diff
@@ -0,0 +1,144 @@
+from __future__ import unicode_literals, division, absolute_import
+import logging
+
+from flexget import plugin
+from flexget.event import event
+from flexget.utils import json
+from flexget.utils.template import RenderError
+from flexget.config_schema import one_or_more
+
+log = logging.getLogger('hipchat')
+
+
+class OutputHipchat(object):
+    """
+    Example::
+      hipchat:
+        auth_token: <AUTH_TOKEN>
+        room_key: <ROOM_KEY>
+        [color: green|yellow|red|grey|purple (default: green)]
+        [notify: true|false (default: false)]
+        [title: <TITLE> (default: "New download started:")]
+        [message: <MESSAGE_BODY> (default: "{{ series_name }} {{ series_id }} {{ quality }}" -- accepts Jinja2)]
+        [url: <HIPCHAT_URL> (default: https://api.hipchat.com)]
+    Configuration parameters are also supported from entries (eg. through set).
+    """
+    default_message = ('{% if series_name is defined %}{{tvdb_series_name|d(series_name)}} {{series_id}} '
+                        '{{tvdb_ep_name|d('')}}{% elif imdb_name is defined %}{{imdb_name}} '
+                        '{{imdb_year}}{% else %}{{title}}{% endif %}')
+    schema = {
+        'type': 'object',
+        'properties': {
+            'auth_token': one_or_more({'type': 'string'}),
+            'room_key': one_or_more({'type': 'string'}),
+            'color': {'type': 'string', 'default': 'green'},
+            'notify': {'type': 'string', 'default': 'false'},
+            'title': {'type': 'string', 'default': 'New download started:'},
+            'message': {'type': 'string', 'default': default_message},
+            'url': {'type': 'string', 'default': 'https://api.hipchat.com'},
+        },
+        'required': ['auth_token', 'room_key'],
+        'additionalProperties': False
+    }
+
+    # Run last to make sure other outputs are successful before sending notification
+    @plugin.priority(0)
+    def on_task_output(self, task, config):
+
+        for entry in task.accepted:
+            auth_token = config['auth_token']
```

Comment:
@liiight yes, but it's always paired, auth_token + room_key and I don't know how it can be forced to be paired.